Simplified/NYPLBookAcquisitionPath.m

```diff
@@ -177,39 +177,54 @@ in mutableTypePaths(nestedIndirectAcquisition, allowedTypes))
   NSMutableSet *const mutableAcquisitionPathSet = [NSMutableSet set];
   NSMutableArray *const mutableAcquisitionPaths = [NSMutableArray array];
 
-  for (NYPLOPDSAcquisition *const acquisition in acquisitions) {
-    if ([types containsObject:acquisition.type]
-        && NYPLOPDSAcquisitionRelationSetContainsRelation(relations, acquisition.relation))
-    {
-      if (acquisition.indirectAcquisitions.count == 0) {
-        [mutableAcquisitionPaths addObject:
-         [[NYPLBookAcquisitionPath alloc]
-          initWithRelation:acquisition.relation
-          types:@[acquisition.type]
-          url:acquisition.hrefURL]];
-      } else {
-        NSMutableSet<NSString *> *supportedSubtypes = [[NYPLBookAcquisitionPath supportedSubtypesForType:acquisition.type] mutableCopy];
+    // Made the function a little more readable.
+    for (NYPLOPDSAcquisition *const acquisition in acquisitions) {
+        BOOL containsType = [types containsObject:acquisition.type];
+        BOOL containsRelation = NYPLOPDSAcquisitionRelationSetContainsRelation(relations, acquisition.relation);
+        BOOL shouldAdd = containsType && containsRelation;
+        
+        if (!shouldAdd) {
+            continue;
+        }
+        
+        if (acquisition.indirectAcquisitions.count == 0) {
+            [mutableAcquisitionPaths addObject:
+             [[NYPLBookAcquisitionPath alloc]
+              initWithRelation:acquisition.relation
+              types:@[acquisition.type]
+              url:acquisition.hrefURL]];
+            continue;
+        }
+        
+        NSMutableSet<NSString *> *supportedSubtypes = [[NYPLBookAcquisitionPath
```

Comment:
this is the same style of formatting I prefer 👍  I agree it's much more readable.